CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 All notable changes to this project will be documented in this file.
 
+---
+## [1.0.0] - 2022-07-27
+### Added
+### Changed
+- Add doc
+- Several object internals have been made private
+- review code for ANSSI submission
+### Fixed
+### Removed
+- `ap()` is replaced by `AccessPolicy::new()`
```

Comment:
```suggestion
- `ap()` is removed. Use `AccessPolicy::new()` instead
```

Reply:
I am pretty sure than `ap` function has been made for practical usage (and to avoid to write `AccessPolicy::new()`)

Reply:
It is possible, but this means having two times the same code, and I don't think this is a good thing. I don't see why calling `AccessPolicy::new()` is not "practical".